stdlib/email/src/main/ballerina/Ballerina.toml

```diff
@@ -0,0 +1,18 @@
+[project]
+org-name = "ballerina"
+version = "1.1.5"
```

Comment:
This version represents the version of the particular std lib module base on its maturity and the functionality coverage. This is totally independent from the ballerina version. 
So If we are confident enough that this module has covered all basic functionalities we can version this as 1.0.0. If not something less would be meaningful.

Reply:
Changed to version 0.9.0.